src/scheduler/worker.rs

```diff
@@ -4,13 +4,13 @@ use super::*;
 use crate::mmtk::MMTK;
 use crate::util::copy::GCWorkerCopyContext;
 use crate::util::opaque_pointer::*;
-use crate::vm::VMBinding;
+use crate::vm::{Collection, GCThreadContext, VMBinding};
 use atomic_refcell::{AtomicRef, AtomicRefCell, AtomicRefMut};
-use std::sync::atomic::{AtomicBool, Ordering};
+use crossbeam::deque::{Stealer, Worker};
```

Comment:
I find the presence of the `Worker` type a bit confusing in the context where we frequently deal with GC workers.  Can we rename it when importing, like `{Worker as CBWorker}` or `{Worker as CrossBeamWorker}`?  Alternatively we can always qualify it as `deque::Worker` when we use it.

Reply:
Agreed. This is qualified to `deque::Worker`.